cmd/gmailctl/cmd/init_cmd.go

```diff
@@ -14,6 +16,8 @@ import (
 var (
 	initReset          bool
 	initRefreshExpired bool
+	initResetLib       bool
+	initRestoreLib     bool
```

Comment:
I would say, restoring is not necessary. We don't do it with `gmailctl init --reset`, so I don't think the library warrants more protections.